.release/pypi/inference.cli.setup.py

```diff
@@ -19,7 +19,7 @@
 
 from inference.core.version import __version__
 
-with open("inference_cli/README.md", "r") as fh:
+with open("inference_cli/README.md", "r", encoding="utf-8") as fh:
```

Comment:
needed this for the wheels to be buildable on windows.  otherwise the emojis in the doc end up throwing encoding errors